src/main/java/org/candlepin/resource/ConsumerResource.java

```diff
@@ -25,91 +30,18 @@
 import org.candlepin.audit.EventBuilder;
 import org.candlepin.audit.EventFactory;
 import org.candlepin.audit.EventSink;
-import org.candlepin.auth.Access;
-import org.candlepin.auth.ActivationKeyPrincipal;
-import org.candlepin.auth.ConsumerPrincipal;
-import org.candlepin.auth.Principal;
-import org.candlepin.auth.SecurityHole;
-import org.candlepin.auth.SubResource;
-import org.candlepin.auth.UpdateConsumerCheckIn;
-import org.candlepin.auth.UserPrincipal;
-import org.candlepin.auth.Verify;
+import org.candlepin.auth.*;
```

Comment:
We don't use star imports (except for static imports in unit test classes), so the checkstyle stage in Jenkins will fail here (see https://github.com/candlepin/candlepin/blob/358b5d0f790265a36591811b1c6f26fada64d122/config/checkstyle/checkstyle.xml#L185-L187)
Same goes for all of the files in the PR

Reply:
yes, Ondrej has already pointed this out to me so I fixed it in the last commit